### ISSUE TITLE
[Kubuntu 17.10] - Segfault when starting after a system update

### ISSUE THREAD

**Issue by Barbosabyte** [psifidotos/Latte-Dock#844]
Hello.
I have exactly the same issue as #788 
There's a difference in some version numbers:

    KDE Plasma: 5.11.5
    KDE Framework: 5.41.0
    Latte-Dock: 0.7.3

Anyway, I tried cleaning the cache and I tried deleting files on ~/.config but it didn't work.

```
Thread 4 "QQmlThread" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 0x7fffd917a700 (LWP 18325)]
0x00007ffff53874d0 in ?? () from /usr/lib/x86_64-linux-gnu/libQt5Qml.so.5
```

**Comment by psifidotos** [comment]
1. When does it crash?
2. It happened after an update or you never managed to open Latte?
3. Qt version?

**Comment by Barbosabyte** [comment]
1 - Crashes on autostart or manual start.
2 - I updated something (I can't really remember what) the day before. I usually shutdown the computer at night, so when I started after the update.
3 - Qt 5.9.1 (I didn't mention it because it was the same as #788 ).

**Comment by psifidotos** [comment]
What distro is still using qt 5.9.1?

**Comment by Barbosabyte** [comment]
Kubuntu 17.10 with backports

**Comment by psifidotos** [comment]
Was latte closed when you cleared the Latte  cache? You check it by using ksysguard

**Comment by psifidotos** [comment]
@Barbosabyte in case you want to provide a more informative backtrace you should try: #147

**Comment by Barbosabyte** [comment]
Here's ddd backtrace:

![screenshot_20180129_182108](https://user-images.githubusercontent.com/7090831/35527086-78b868ec-0521-11e8-8d74-16a21adf5cee.png)

Running `env QV4_FORCE_INTERPRETER=1 latte-dock -d` makes the dock work. 
Running normally still segfaults.

**Comment by psifidotos** [comment]
> Running env QV4_FORCE_INTERPRETER=1 latte-dock -d makes the dock work. Running normally still segfaults.

unfortunately the backtrace isnt any helpful, it just points to QML. the fact that works for `env QV4_FORCE_INTERPRETER=1` it points a bit that this is a cache issue.

**Comment by Barbosabyte** [comment]
I tried cleaning the cache again. (with latte closed)
I tried removing and reinstalling (cleaning the cache again while removed).
Didn't work

**Comment by psifidotos** [comment]
Sorry I dont have any other ideas... I suppose more Kubuntu users must get into the discussion and help trace it down if it occurs also to their systems...

**Comment by psifidotos** [comment]
If you close Latte and remove totally the following items:

folder: `~/.config/latte`
and file: `~/.config/lattedockrc`

in order to start Latte with default settings does it crash again?

**Comment by Barbosabyte** [comment]
I purged latte from my system. Had to manually remove the cache and config files.
I had a warning from dpkg and had to manually remove
`/usr/share/plasma/shells/org.kde.latte.shell/` and 
`/usr/share/plasma/plasmoids/org.kde.latte.plasmoid/` folders.
Then I reinstalled. It ran with default settings.
I had a backup of the config files, so I put them back and it still works.
Don't know if it really was a cache issue or something related to those two folders.
Closing the issue as it is solved.
